docs/install-1.7.md

```diff
@@ -143,7 +143,8 @@ helm install ../../charts/catalog \
         --set useAggregator=true \
         --set apiserver.tls.ca=$(base64 --wrap 0 ${SC_SERVING_CA}) \
         --set apiserver.tls.cert=$(base64 --wrap 0 ${SC_SERVING_CERT}) \
-        --set apiserver.tls.key=$(base64 --wrap 0 ${SC_SERVING_KEY})
+        --set apiserver.tls.key=$(base64 --wrap 0 ${SC_SERVING_KEY}) \
+        --set apiserver.tls.requestHeaderCA=$(base64 ${SC_SERVING_CA})
```

Comment:
Should this last command use `--wrap` like the others?

Reply:
yes, good catch